AppCenterAnalytics/AppCenterAnalytics/MSAnalytics.m

```diff
@@ -369,9 +384,19 @@ - (void)trackPage:(NSString *)pageName withProperties:(NSDictionary<NSString *,
 }
 
 - (void)sendLog:(id<MSLog>)log flags:(MSFlags)flags {
+  [(flags & MSFlagsCritical) != 0 ? self.criticalChannelUnit : self.channelUnit enqueueItem:log flags:flags];
+}
 
-  // Send log to log manager.
-  [self.channelUnit enqueueItem:log flags:flags];
+- (void)setTransmissionInterval:(NSUInteger)interval {
+  if (self.started) {
+    MSLogError([MSAnalytics logTag], @"The transmission interval should be set before the MSAnalytics service is started.");
+    return;
+  }
+  if (interval > kMSFlushIntervalMaximum || interval < kMSFlushIntervalMinimum) {
+    MSLogError([MSAnalytics logTag], @"The transmission interval is not valid, it should be between 3 seconds and 1 day (86400 seconds).");
```

Comment:
Use constants instead of 3 seconds and 1 day in logs. There was a comment about that in android PR  https://github.com/microsoft/appcenter-sdk-android/pull/1145#discussion_r282121071